simphony/cuds/abc_dataset.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+from abc import ABCMeta, abstractmethod
+
+
+class ABCDataset(object):
```

Comment:
DataSets must conform the the structore of `CUBA.DATA_SET`, i.e. they need `uid`, `name`, etc defined.